ingenialink/ethernet/dictionary.py

```diff
@@ -1,7 +1,10 @@
+from typing import Optional, Dict, List
+
 from ingenialink.dictionary import Dictionary
 from ingenialink.ethernet.register import EthernetRegister
 
 import ingenialogger
+import xml.etree.ElementTree as ET
```

Comment:
Imports orders should be:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

https://peps.python.org/pep-0008/#imports

Reply:
Done!